sitecustomize.py

```diff
@@ -0,0 +1,9 @@
+"""Setup coverage tracking for subprocesses.
+
+Any ImportError is silently ignored.
+Requires COVERAGE_PROCESS_START in the environments, which gets set in
+tox.ini.
+"""
+import coverage
+
+coverage.process_startup()
```

Comment:
I've used `coverage-enable-subprocess` with success before -- it essentially does this but with a `.pth` file (no need for a relative `sitecustomize.py` file)

Reply:
Awesome, will use that.